workers/Adarsh-NP/database_demo/commitdata_collection.py

```diff
@@ -0,0 +1,33 @@
+import os, json, requests
```

Comment:
I think we don't want this file. The Adarsh-NP folder should be deleted from your branch.